### ISSUE TITLE
Cannot start web server in version 2.5

### ISSUE THREAD

**Issue by pbienst** [mnemosyne-proj/mnemosyne#7]
From: https://bugs.launchpad.net/mnemosyne-proj/+bug/1717810

Whenever I try to run the web server I get the following error:

AttributeError("'NoneType' object has no attribute 'set_render_chain'",)
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/cheroot/server.py", line 1124, in communicate
    req.respond()
  File "/usr/local/lib/python3.6/dist-packages/cheroot/server.py", line 951, in respond
    self.server.gateway(self).respond()
  File "/usr/local/lib/python3.6/dist-packages/cheroot/wsgi.py", line 125, in respond
    response = self.req.server.wsgi_app(self.env, self.start_response)
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/web_server/web_server.py", line 145, in wsgi_app
    self.load_mnemosyne()
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/web_server/web_server.py", line 111, in load_mnemosyne
    filename=self.filename, automatic_upgrades=False)
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/libmnemosyne/__init__.py", line 237, in initialise
    self.start_review()
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/libmnemosyne/__init__.py", line 362, in start_review
    self.config()["study_mode"]))
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/libmnemosyne/controllers/default_controller.py", line 100, in set_study_mode
    study_mode.activate()
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/libmnemosyne/study_modes/scheduled_forgotten_new.py", line 22, in activate
    self.activate_components()
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/libmnemosyne/study_mode.py", line 27, in activate_components
    Component.activate(self) # Also activates GUI components.
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/libmnemosyne/component.py", line 74, in activate
    component = component(component_manager=self.component_manager)
  File "/usr/local/lib/python3.6/dist-packages/Mnemosyne-2.5-py3.6.egg/mnemosyne/web_server/review_wdgt.py", line 26, in __init__
    self.review_controller().set_render_chain("web_server")
AttributeError: 'NoneType' object has no attribute 'set_render_chain'

I get this error on both Ubuntu and Arch Linux.

**Comment by pbienst** [comment]
Seems to have already been fixed by Julian's previous patches.

**Comment by DorianHawkmoon** [comment]
I don't think this is actually fixed. With version 2.6 in Archlinux I get the same error message. Same with the git version. I cloned the git repository, make build-all-deps, make run.
Added a card (just in case), went to Settings, Servers and activated "Allow remote review through a web browser".
Exactly in the moment I try to connect to the given url through vivaldi or firefox, the error appears on console and a little window with the message "Remote review in progress...". 

I attached the output for the git version.
[output.txt](https://github.com/mnemosyne-proj/mnemosyne/files/1629831/output.txt)
